handlers/tweet.js

```diff
@@ -0,0 +1,330 @@
+ *                  user_id:
+ *                     type: integer
+ *                  text:
+ *                     type: string
+ *                  photo:
+ *                     type: string
+ */
+tweet.get('/tweets/:id/tweet',
+    validate([param('id').isInt({ min: 1 })]),
+    async (request, response) => {
+        try {
+            const tweet = await tweetController.getTweet(request.params.id);
+            response.json(tweet);
+        } catch (error) {
+            request.log.error(error);
+            response.end(JSON.stringify(error)).status(500);
+        }
+    });
+
+/**
+ * @openapi
+ * /tweets/{id}/tweets:
+ *   get:
+ *     description: Returns all tweets by a specific user.
+ *     tags:
+ *      - Tweet
+ *     parameters:
+ *       - in: path
+ *         name: id
+ *         description: user id
+ *         schema:
+ *          type: integer
+ *     responses:
+ *       200:
+ *        description: Returns all tweets by a specific user.
+ *        content:
+ *          application/json:
+ *             schema: 
+ *                type: object
+ *                properties:
+ *                  id:
+ *                     type: integer
+ *                  user_id:
+ *                     type: integer
+ *                  text:
+ *                     type: string
+ *                  photo:
+ *                     type: string
+ */
+tweet.get('/tweets/:id/tweets',
+    validate([param('id').isInt({ min: 1 })]),
+    async (request, response) => {
+        try {
+            const tweets = await tweetController.getTweetByUser(request.params.id);
+            response.json(tweets);
+        } catch (error) {
+            request.log.error(error);
+            response.end(JSON.stringify(error)).status(500);
+        }
+    });
+
+
+/**
+ * @openapi
+ * /users/tweets:
+ *   post:
+ *     description: Tweet.
+ *     tags:
+ *      - Tweet
+ *     produces:
+ *       - application/json
+ *     requestBody:
+ *       content:
+ *         application/json:
+ *          schema:
+ *           type: object   
+ *           properties: 
+ *                  text:
+ *                     type: string
+ *                  photo:
+ *                     type: string
+ *     security:
+ *      - bearerAuth: [] 
+ *     responses:
+ *       200:
+ *        description: Tweet.
+ *        content:
+ *          application/json:
+ *             schema: 
+ *                type: object
+ *                properties:
+ *                  id:
+ *                     type: integer
+ *                  user_id:
+ *                     type: integer
+ *                  text:
+ *                     type: string
+ *                  photo:
+ *                     type: string
+ *       404:
+ *          description: Impossible to tweet.
+ *          content:
+ *            application/json:
+ *              schema: 
+ *                type: string
+ */
+tweet.post('/users/tweets', authentication, validate([
```

Comment:
Here /tweets would be enough, it's clear you are creating a tweet, and the required user_id is included in the request body or taken from the token. Remove the /users